$MirOS: ports/multimedia/mplayer/patches/patch-libdha_Makefile,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
$OpenBSD: patch-libdha_Makefile,v 1.13 2004/05/01 18:43:43 sturm Exp $
--- libdha/Makefile.orig	Sat Jan  1 14:44:49 2005
+++ libdha/Makefile	Thu May 19 20:35:14 2005
@@ -53,8 +53,6 @@ ifeq ($(TARGET_WIN32),yes)
 else
 	#$(CC) -shared $(SONAME_FLAGS) -o $(LIBNAME) $(OBJS) $(LIBS)
 	$(CC) -shared -Wl,-soname -Wl,$(LIBNAME) -o $(LIBNAME) $(OBJS) $(LIBS)
-	ln -sf $(LIBNAME) $(SHORTNAME)
-	ln -sf $(LIBNAME) $(VSHORTNAME)
 endif
 
 all:    $(LIBNAME) $(SHORTNAME)
@@ -80,12 +78,12 @@ depend: pci_names.c
 install:
 	mkdir -p $(LIBDIR)
 	install -m 755 -s -p $(LIBNAME) $(LIBDIR)/$(LIBNAME)
-	rm -f $(LIBDIR)/libdha.so
-	ln -sf $(LIBNAME) $(LIBDIR)/$(SHORTNAME)
 ifeq ($(TARGET_OS),OpenBSD)
-	ldconfig -R
 else
+ifeq ($(TARGET_OS),MirBSD)
+else
 	ldconfig
+endif
 endif
 
 uninstall:
